sdk/tests/accountCompression.test.ts

```
@@ -853,8 +297,8 @@ describe('Account Compression', () => {
             payerInfo = await provider.connection.getAccountInfo(payer, 'confirmed')!;
             const finalLamports = payerInfo!.lamports;
             assert(
-                finalLamports === payerLamports + treeLamports - 5000,
-                'Expected payer to have received the lamports from the closed tree account',
+                finalLamports === payerLamports + treeLamports - 1_400_000 * 10_000 / 1_000_000 - 5000,
+                'Expected payer to have received the lamports from the closed tree account, received ' + finalLamports
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)